testnet/tools/ipv6-calc.py

```diff
@@ -77,9 +77,9 @@ def calc_mac_address(serial: str, deployment_name: str, node_index: str):
     return ansible_random_mac("52:00", f"{serial} {deployment_name} {node_index}")
 
 
-def ipv6_address_calculate_slaac(ipv6_prefix: str, ipv6_subnet: str, mac_address: str):
+def ipv6_address_calculate_slaac(ipv6_prefix: str, ipv6_prefix_length: str, mac_address: str):
```

Comment:
Note: If we go much further into any of this, we should try to replace the IP generation code with [deterministic_ips](https://github.com/dfinity/ic/tree/1ddd82018cfcc02b0fdb900a4379339f4e63048b/rs/ic_os/dev_test_tools/deterministic_ips).